turbo/cli/flags.go

```diff
@@ -76,6 +77,14 @@ var (
 	Example: --prune=htc`,
 		Value: "disabled",
 	}
+	MinimalHistoryFlag = cli.BoolFlag{
+		Name:  "minimal-history",
```

Comment:
Maybe can re-use existing CLI flag: like `--prune.minimal=true` or `--prune=minimal`. 
Because in `e2` prune terminology `prune history` means prune history of state - not including trace index, log index, blocks. so `--minimal-history` may be confusing (because this flag now enabling "prune everything to the moon").